PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/ValidationUtils.java

```diff
@@ -44,4 +51,22 @@ public static boolean isValidFunctionName(String name) {
     public static boolean isValidVersion(String version) {
         return version != null && version.matches(VERSION_REGEX);
     }
+
+    public static void validFunctionAppName(String subscriptionId, String functionAppName) {
```

Comment:
'valid' is not a verb, use checkNewFunctionAppName

Reply:
Fixed